package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@shardlabs/starknet-hardhat-plugin",
-    "version": "0.6.2",
+    "version": "0.6.3-alpha.2",
```

Comment:
@Nathan-SL @FabijanC you agreed on using this tag out of the 3 mentioned options? I'm asking because the PR is approved!

Reply:
The version that is currently on branch `proxy-script` (the branch to be merged with this PR) is the one that was released as `0.6.3-alpha.2`